File: Backend/src/ProEventos.Domain/Speaker.cs
using System.Collections.Generic;
using ProEventos.Domain.Identity;

namespace ProEventos.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Curriculum { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public IEnumerable<SocialNetwork> SocialNetworks { get; set; }
        public IEnumerable<SpeakerEvent> SpeakersEvents { get; set; }
    }
}

File: Frontend/ProEventos-App/src/app/models/Speaker.ts
import { SocialNetwork } from "./SocialNetwork";
import { SpeakerEvent } from "./SpeakerEvent";

export interface Speaker {
  id: number;
  name: string;
  curriculum: string;
  imageUri: string;
  phone: string;
  email: string;
  socialNetworks: SocialNetwork[];
  speakersEvents: SpeakerEvent[];
}
